app/views/admin/bulk_imports/show.html.haml

```diff
@@ -5,11 +5,12 @@
       %em
         \##{@bulk_import.id}
   .col-sm-8
-    %p
-      View #{link_to "all bulk imports", admin_bulk_imports_path}
-      - if @bulk_import.organization_id.present?
-        %em
-          \- View #{link_to "all #{@bulk_import.organization.name} imports", admin_bulk_imports_path(organization_id: @bulk_import.organization.to_param)}
+    %ul
+      %li.nav-item
+        View #{link_to "all bulk imports", admin_bulk_imports_path}
```

Comment:
Does this look weird if you add the `nav-link` class?

Reply:
Mmm, not really. It looks pretty much the same just a little slantier. It kinda seems like nav-link almost cancels out %em

![UNADJUSTEDNONRAW_mini_47](https://user-images.githubusercontent.com/20424916/58289491-33346800-7d7c-11e9-94a9-92c5801cdad5.jpg)

I honestly think it might look better without the `%em` but obviously I'll defer to you. It looks like the two links are different for no actual reason.

Reply:
Yeah, no need for `%em`